staging/src/k8s.io/apiserver/pkg/storageversion/updater.go

```diff
@@ -79,6 +79,10 @@ func updateStorageVersionFor(c Client, apiserverID string, gr schema.GroupResour
 
 func singleUpdate(c Client, apiserverID string, gr schema.GroupResource, encodingVersion string, decodableVersions []string) error {
 	shouldCreate := false
+	// Group is a valid subdomain in DNS (RFC 1123).
+	if len(gr.Group) == 0 {
+		gr.Group = "core"
```

Comment:
This will hide the bug if a non-core group doesn't fill the gr.Group. Can you fix the group name where gr is constructed?

Reply:
Good point. I dropped this commit. PTAL